### PR TITLE
Added and modified a rule

### PR DESCRIPTION
**Pull request by hiaashna**
Updated the `rule` -> `sinks` -> `leakages` -> `logs` -> `java.yaml` --> `Leakages.Log.Error` to include `wtf` along with `error|severe|fatal`

Create a new rule in `rule` -> `sinks` -> `leakages` -> `file` -> `java.yaml` where functions which write to a file are marked as `sink`

### CODE REVIEW

**Review by abhstabs — DISMISSED** [review]

File: rules/sinks/leakages/logs/java.yaml [hunk 1]
```diff
@@ -29,3 +29,9 @@ sinks:
     patterns: 
       - "(?i)(java.io.PrintStream|android.util.(?:Log){0,1}Printer).(?:print|println|write).*"
     tags:
+
+  - id: Leakages.Log.Equals
```
Comment (khemrajrathore) [inline rules/sinks/leakages/logs/java.yaml:33]: @hiaashna I guess this rule is not needed, as per my understanding this is used to compare 2 log levels. What you feel?

Comment (hiaashna) [reply rules/sinks/leakages/logs/java.yaml:33]: @khemrajrathore  yeah, I feel the same and I think whichever logger is supposed to expose any personal info have already been covered.

Comment (hiaashna) [reply rules/sinks/leakages/logs/java.yaml]: @khemrajrathore I have removed log equals and added a log that contain IP address

**Review by abhstabs — APPROVED** [review]
LGTM

**Comment by khemrajrathore** [comment]
We will merge and release once `performance` changes are through

**Review by khemrajrathore — APPROVED** [review]
LGTM

**Pull request merged**